### PR TITLE
Added the references to security vulnerabilities discovered in 2016

### PR DESCRIPTION
**Pull request by javiereguiluz**
Dear Symfony Docs maintainers, please give this PR top priority when merging PRs. Thanks!

### CODE REVIEW

File: contributing/code/security.rst [hunk 1]
```diff
@@ -103,8 +103,11 @@ Security Advisories
 This section indexes security vulnerabilities that were fixed in Symfony
 releases, starting from Symfony 1.0.0:
 
-* November 23, 2015: `CVE-2015-8125: Potential Remote Timing Attack Vulnerability in Security Remember-Me Service <http://symfony.com/blog/cve-2015-8125-potential-remote-timing-attack-vulnerability-in-security-remember-me-service>`_ (2.3.35, 2.6.12 and 2.7.7)
```
Comment (xabbuh) [inline contributing/code/security.rst]: @javiereguiluz Any idea why this is shown as removed?

Comment (javiereguiluz) [inline contributing/code/security.rst]: Yes! I'm sorry. I replaced `http://` by `https://` in the links of November 2015 (they were the only ones without HTTPS). Too bad the `diff` algorithm used by GitHub doesn't detect that.

Comment (xabbuh) [inline contributing/code/security.rst]: Oh thanks, I missed that. :(

**Comment by xabbuh** [comment]
👍 

Status: Reviewed

**Comment by xabbuh** [comment]
Thank you Javier.

**Pull request merged**